test/config/integration/echo_server.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Since you are moving this test into a standalone config, is there any reason to leave the echo listener in the other config? Seems like that should be removed?